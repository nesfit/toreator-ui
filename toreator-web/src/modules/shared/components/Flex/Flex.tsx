import styled from "styled-components";
import {alignItems, flexDirection, flexWrap} from "styled-system";
import Box from "../Box/Box";

const Flex: any = styled(Box)`
  display: flex;
  ${flexDirection};
  ${flexWrap};
  ${alignItems};
`;

export default Flex;
