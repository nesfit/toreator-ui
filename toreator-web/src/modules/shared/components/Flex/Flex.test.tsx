import {shallow} from "enzyme";
import * as React from "react";
import Flex from "./Flex";

describe("<Flex/>", () => {
  test("default", () => {
    const wrapper = shallow(<Flex />);
    expect(wrapper).toMatchSnapshot();
  });
});
