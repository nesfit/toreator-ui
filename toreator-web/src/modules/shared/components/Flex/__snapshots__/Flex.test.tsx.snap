// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Flex/> default 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": false,
        "rules": Array [
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  ",
          [Function],
          "
  box-sizing: border-box;
  ",
          [Function],
          "  
  ",
          [Function],
          "  
  ",
          [Function],
          "  
",
          "
  display: flex;
  ",
          [Function],
          ";
  ",
          [Function],
          ";
  ",
          [Function],
          ";
",
        ],
      },
      "displayName": "Styled(styled.div)",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
